feconf.py

```diff
@@ -637,7 +637,7 @@ def get_empty_ratings():
 SHOW_COLLECTION_NAVIGATION_TAB_STATS = False
 
 # Bool to enable update of analytics models.
-ENABLE_NEW_STATS_FRAMEWORK = False
+ENABLE_NEW_STATS_FRAMEWORK = True
```

Comment:
You shouldn't do this. This messes up every other test. You should change this at only the places where you need it. The way I see it, you don't even need this to be True for the jobs above. Why did you do this?

Reply:
This is only turned on for now so I could do some manual testing, it's not meant to be on the actual PR.

Reply:
@LanJosh Okay. Make sure this is removed later. I'll do a review on this PR in a few hours!